docs/status.md

```diff
@@ -114,7 +114,7 @@ one of the following tracking labels.
 | pad                      | yes           | yes          | yes            | yes             | no          |
 | partition_id             | yes           | yes          | yes            | yes             | no          |
 | popcnt                   | yes           | yes          | yes            | yes             | no          |
-| power                    | yes           | revisit      | yes            | yes             | no          |
+| power                    | yes           | yes          | yes            | yes             | no          |
 | real                     | yes           | yes          | yes            | yes             | no          |
 | real_dynamic_slice       | no            | revisit      | no             | yes             | no          |
 | recv                     | yes           | revisit      | infeasible     | no              | no          |
```

Comment:
Looks like the Verification column of `shift_left`, `shift_right_arithmetic` and `shift_right_logical` needs to be flipped to "yes" as well.